packages/params/src/forkName.ts

```diff
@@ -110,11 +110,7 @@ export function isForkPostFulu(fork: ForkName): fork is ForkPostFulu {
   return isForkPostElectra(fork) && fork !== ForkName.electra;
 }
 
-export type ForkPreBlobs = ForkPreDeneb;
-export type ForkPostBlobs = ForkPostFulu;
-export type ForkBlobs = ForkName.deneb | ForkName.electra;
-export const forkBlobs = [ForkName.deneb, ForkName.electra];
-export function isForkBlobs(fork: ForkName): fork is ForkBlobs {
+export function isForkBlobs(fork: ForkName): fork is ForkName.deneb | ForkName.electra {
```

Comment:
why is this still a think? I though we decided to not modify `isForkBlobs` and have a specific fork guard just for the DA code

this seems unused as well, let's keep it as an alias for `isForkPostDeneb` as it's done on unstable branch
https://github.com/ChainSafe/lodestar/blob/6fe1606bdbcb80039d29f3a3613883bf4ff6fbe9/packages/params/src/forkName.ts#L143-L146

Reply:
@matthewkeil see https://github.com/ChainSafe/lodestar/pull/6353#discussion_r2258278473 how to better structure our else-if statements handling different forks